DESCRIPTION

```diff
@@ -10,6 +10,7 @@ Authors@R: c(
   person("Bruno", "Tremblay", role = "ctb", email = "cran@neoxone.com"),
   person("Frans", "van Dunné", role = "ctb", email = "frans@ixpantia.com"),
   person("Sebastiaan", "Vandewoude", role="ctb", email = "sebastiaanvandewoude@gmail.com"),
+  person("Josiah", "Parry", role = "ctb", email = "josiah.parry@gmail.com", comment = c(ORCID = "0000-0001-9910-865X")),
```

Comment:
Could you add a NEWS entry and tag your GitHub handle with the PR number? Thank you!

```suggestion
```